src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -33,6 +33,7 @@
         <parameter key="translation.loader.class">Symfony\Bundle\FrameworkBundle\Translation\TranslationLoader</parameter>
         <parameter key="translation.extractor.class">Symfony\Component\Translation\Extractor\ChainExtractor</parameter>
         <parameter key="translation.writer.class">Symfony\Component\Translation\Writer\TranslationWriter</parameter>
+        <parameter key="translation.warmer.class">Symfony\Bundle\FrameworkBundle\CacheWarmer\TranslationsCacheWarmer</parameter>
```

Comment:
Symfony used to use class parameters for a long time to make the class names configurable. Though there are better ways to modify services so that it was decided to not promote these parameters anymore and also don't introduce new ones (all services introduced with Symfony 2.6 don't declare them). The existing parameters have to be kept for BC, but they will be removed in Symfony 3.0.